test/e2e/helpers/SauceLabsReportingHelper.js

```diff
@@ -9,22 +9,26 @@ const sauceUsername = process.env.SAUCE_USERNAME || config.get('saucelabs.userna
 const sauceKey = process.env.SAUCE_ACCESS_KEY || config.get('saucelabs.key');
 
 
-function updateSauceLabsResult(result, sessionId) {
+function updateSauceLabsResult(result, sessionId, jobName) {
   const sauceUrl = `https://saucelabs.com/rest/v1/${sauceUsername}/jobs/${sessionId}`;
   const sauceCredentials = `-u ${sauceUsername}:${sauceKey}`;
+  // For publishing SauceLabs results through Jenkins Sauce OnDemand plugin:
+  console.log(`SauceOnDemandSessionID=${sessionId} job-name=${jobName}`);
```

Comment:
cannot you use a logger instead like here -> https://github.com/hmcts/sscs-submit-your-appeal/blob/760e75a8ca38765b707f5c8206056edddc3ae6d4/steps/start/postcode-checker/BranchForEnglandOrWales.js#L3 ? :)

Reply:
good call, I've added it now - still works with the Jenkins plugin too (which is nice!)